### DISCUSSION TITLE
New integration for Candy fridge cbl3518evw (hOn app)

### DISCUSSION THREAD

**Discussion by andrearomanoif** [ofalvai/home-assistant-candy#212] [Device support improvements]
Hello everyone. I have a candy cbl3518evw fridge connected through the app "hOn" distributed also for Hover connected products. Would an implementation be possible for the management of this fridge? I can also be of support and collaborate on a possible development used my fridge.

**Comment by ofalvai** [comment]
Hello @andrearomanoif 👋 

Sorry for the late reply. We can give it a try, please see [this section](https://github.com/ofalvai/home-assistant-candy#my-device-isnt-supported-can-you-help) about how to add support for a new device.
